### PR TITLE
Bugfix/MYR-384: Fix walletAddress should be associated with postId on send tip

### PR DESCRIPTION
**Pull request by ical10**

### CODE REVIEW

File: src/components/post/post.component.tsx [hunk 1]
```diff
@@ -82,6 +98,9 @@ export default function PostComponent({
       return;
     }
 
+    console.log('the sendTipRef: ', sendTipRef);
```
Comment (irmannmal) [inline src/components/post/post.component.tsx:101]: bor @ical10  log ini bisa di ilangin dulu ga

File: src/components/profile/myWallet/transaction.component.tsx [hunk 2]
```diff
@@ -321,10 +310,13 @@ const TransactionComponent: React.FC<TransactionProps> = ({ forwardedRef, detail
     state: { user }
   } = useUser();
   const userAddress = session?.user.address as string;
+  console.log('userAddress is: ', userAddress);
```
Comment (irmannmal) [inline src/components/profile/myWallet/transaction.component.tsx:313]: ini juga bor @ical10 log nya kalau bisa ilangin dulu

File: src/components/post/post.component.tsx (hunk 1 above)
Comment (ical10) [reply src/components/post/post.component.tsx:101]: resolved di next commit ya mas

File: src/components/profile/myWallet/transaction.component.tsx (hunk 2 above)
Comment (ical10) [reply src/components/profile/myWallet/transaction.component.tsx:313]: resolved di next commit ya mas

**Review by irmannmal — APPROVED** [review]

**Pull request merged**